frontend/src/components/CredExList.vue

```diff
@@ -286,10 +293,12 @@ export default {
   props: {
     isActiveFn: {
       type: Function,
-      default: (item) =>
-        item.state === CredentialExchangeStates.CREDENTIAL_ISSUED ||
-        item.state === CredentialExchangeStates.CREDENTIAL_ACKED ||
-        item.state === CredentialExchangeStates.DONE,
+      default: (item: CredEx) =>
```

Comment:
None of the types in backend-types can be used as values (unless you know a way) - therefore the constants are still necessary.
Again here is the issue that the backend returns the state as lowercase but defines it as uppercase. The constant object is defined as lowercase already so the toLowercase() is not correct here anyways (it just works because it converts the enum values to the string type. I will change it to "as string" to preserve the type.
As for the backend lowercase content, we might have to do something about it.

Reply:
OK I get it the issue is that the enums have no values, but I think this is also an issue of the generator plugin. For example: https://www.npmjs.com/package/openapi-typescript-codegen seems to work as expected, it generates:

```
enum Foo {
  ONE = "ONE"
}
```
we get 
```
enum Foo {
  ONE
}
```

This is of course not usable. In the first case I would also expect that the types are deserialised correctly when they are returned from axios, no matter if they are lower or upper case. Like as it is now enums are not type safe at all, they are simply string containers. So I see the following options:

1. try other openapi generator plugins
2. add an equals function to the types in constant.ts
3. switch backend serialisation to upper case

1 would be my preferred way, because then we could use the generated types everywhere.